package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "atxfloodsadmin",
```

Comment:
If we go with the super-repo approach do you think we should go ahead and move this into a `admin/` folder or whatever in the project root now?